middleware/localNamingConventions.js

```diff
@@ -0,0 +1,39 @@
+
+function setup() {
+  return applyLocalNamingConventions;
+}
+
+function applyLocalNamingConventions(req, res, next) {
+
+  // do nothing if no result data set
+  if (!res || !res.data) {
+    return next();
+  }
+
+  // loop through data items and flip relevant number/street
+  res.data.filter(function(place){
+    // only relevant for German addresses
+    if( 'DEU' !== place.alpha3 ){ return false; }
```

Comment:
fair point, would you prefer to block this release to future-proof the functionality or be more 'lean' and wait to do it when required? ;)